cogs/admin.py

```diff
@@ -74,23 +71,35 @@ async def sharedservers(self, ctx, *, user: converters.GlobalUser):
                 else:
                     page.description += f"\n{guild}"
             page.set_footer(text="Use the reactions to flip pages.")
-            all_pages.append(page)
+            all_pages.append(page.to_dict())
         if len(all_pages) == 1:
-            embed = all_pages[0]
+            embed = discord.Embed.from_dict(all_pages[0])
             embed.set_footer(text=discord.Embed.Empty)
             await ctx.send(embed=embed)
             return
-        paginator = Paginator(length=1, entries=all_pages, use_defaults=True, embed=True, timeout=120)
-        await paginator.start(ctx)
+        await self.bot.create_reaction_menu(ctx, all_pages)
+
+    async def invite_guild(self, guild_id):
+        guild = await self.bot.get_guild(guild_id)
+        if not guild:
+            return
+        try:
+            invite = (await guild.invites())[0]
+        except (IndexError, discord.Forbidden):
+            try:
+                invite = await self.bot.http.create_invite((await guild.text_channels())[0].id, max_age=120)
+            except discord.Forbidden:
+                return
+        return invite
 
     @checks.is_admin()
     @commands.command(
         description="Create an invite to the specified server.",
         usage="createinvite <server ID>",
         hidden=True,
     )
-    async def createinvite(self, ctx, *, guild: converters.GlobalGuild):
-        invite = await self.bot.comm.handler("invite_guild", -1, {"guild_id": guild.id})
+    async def createinvite(self, ctx, *, guild: int):
```

Comment:
What's the difference? Currently you also have to pass in an ID.

Reply:
It's different because the converter checks that the ID is valid and converts it into a guild object. By your logic, we can simply accept string for everything.